packages/@react-aria/autocomplete/docs/useSearchAutocomplete.mdx

```diff
@@ -69,7 +69,7 @@ Autocomplete for search fields can be implemented using the [&lt;datalist&gt;](h
 
 ## Anatomy
 
-<img src={anatomy} alt="Anatomy of useSearchAutocomplete" height={400} style={{display: 'block', margin: '20px auto'}} />
+<Anatomy />
```

Comment:
You'll want to add `role="img"` and perserve the alt as an `aria-label` (and maybe with a little more detail to what the diagram is displaying). We actually need to do this for a lot of our anatomy diagrams haha.

Perfectly fine with this being a follow up/later task if you want since these docs aren't currently exposed